### ISSUE TITLE
Accessibility/labelling issues

### ISSUE THREAD

**Issue by GBKS** [muun/falcon#23]
Did some quick accessibility testing on iOS using Voice Control and VoiceOver, and stumbled across a few issues. The major one is that users relying on non-touch input cannot access their transaction history at all, as the button is not selectable. The other ones are minor and fixing them will help users better understand what is on the screen and what actions they can take.

![Muun iOS app accessibility feedback](https://user-images.githubusercontent.com/695901/176107871-3c500781-6880-4b50-9f06-a1f772d9b18c.png)

My test workflow is shown in [this video](https://www.youtube.com/watch?v=n9FvI5SfpoU&t=4s) (for a different app, but the workflow is the same). This was a pretty quick and casual test, I did not thoroughly go over every single screen.

**Comment by esneider** [comment]
Great findings! Thanks for taking the time to do the testing Christoph 🙏

We'll add these fixes to the backlog.

**Comment by GBKS** [comment]
Going to close this issue as it's been inactive for 4 years. Might be a good time for someone to do a new round of testing. In the times of AI, getting accessibility right should be very straightforward.
